src/client/mapblock_mesh.cpp

```diff
@@ -695,9 +698,16 @@ static u8 face_contents(content_t m1, content_t m2, bool *equivalent,
 			return 2;
 	}
 
-	if (c1 > c2)
+	if (c1 > c2) {
+        if (enable_waving_water && f2.isLiquid() && tid1 == 1) {
+            return 2;
+        }
```

Comment:
> Do not unnecessarily use braces where a single statement will do.

https://www.kernel.org/doc/Documentation/process/coding-style.rst
https://dev.minetest.net/Code_style_guidelines

Reply:
The 3 lines above seem to have weird indentation, only use tabs not spaces.